src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,6 @@
-const booleanToWord = (boolean) => {};
-
+const booleanToWord = (boolean) => {
+    return (boolean.toLowerCase()==='yes')? true: (boolean.toLowerCase==='no')? false:undefined;
+};
```

Comment:
This works with the tests youve written, but this isnt what this was intended for. See the notes on the test file for details. To fix it you could use a _single_ ternary operator:
```
  return (boolean === true) ? "yes" : "no";
```
To go a little bit further, this could all be put on one line like this:
```
const booleanToWord = (boolean) => (boolean === true ? "yes" : "no");
```
This is the exact same as above, as when putting a function expression on one line like this you no longer need to use the `return` keyword as it is automatically implied